website/docs/d/web_pubsub_hub.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
if we are going to create this data source could we add all the properties instead of leaving it barebones like this?